src/test/index.ts

```diff
@@ -0,0 +1,18 @@
+import { setupServer } from '../server';
+import axios from 'axios';
+import chai from 'chai';
```

Comment:
```suggestion
import { expect } from 'chai';
```

Assim vc evita de ficar chamando `chai.expect` toda a hora.